app/views/layouts/main.html.erb

```diff
@@ -24,8 +24,8 @@
   <%- if flash[:notice] -%>
     <p class="notice"><%= flash[:notice] %></p>
   <%- end -%>
-  <%- if flash[:error] -%>
-    <p class="error"><%= flash[:error] %></p>
+  <%- if flash[:error] or flash[:alert]-%>
+    <p class="error"><%= flash[:error] %><%= flash[:alert] %></p>
```

Comment:
when I issued devise:install to initialize it said that it needed to handle notice and alert flashes from application.html.erb. But maybe in this usage of devise it is not used. I'll check this and remove it.